kbfsmd/root_metadata_v3.go

```diff
@@ -208,12 +208,16 @@ func (extra ExtraMetadataV3) IsReaderKeyBundleNew() bool {
 // must be done separately.
 func MakeInitialRootMetadataV3(tlfID tlf.ID, h tlf.Handle) (
 	*RootMetadataV3, error) {
-	if tlfID.Type() != h.Type() {
+	switch {
+	case h.TypeForKeying() == tlf.TeamKeying && tlfID.Type() != tlf.SingleTeam:
```

Comment:
@strib I assumed this is what you meant; but let me know if I'm wrong!

Reply:
I think this is wrong, because a private, implicit-teamed-backed folder would fail this test.  What I meant was:

```go
case h.TypeForKeying() == tlf.TeamKeying && tlfID.Type() == tlf.SingleTeam && h.Type() != tlf.SingleTeam:
    fallthrough
```

Reply:
Ah that's right. Clearly my body's still in west coast time!